modules/ROOT/pages/authentication-authorization/built-in-roles.adoc

```diff
@@ -1,68 +1,76 @@
-[role=enterprise-edition]
+[role=enterprise-edition aura-db-enterprise]
 [[auth-built-in-roles]]
-= Built-in roles
-:description: This page describes the roles that come pre-defined with Neo4j. 
+= Built-in roles and privileges
```

Comment:
as noted on the operations manual PR, the `LOAD ON ALL DATA` PRs will also need porting  (both the operations manual one and the cypher manual one)

Reply:
This also belongs to the group of comments left open to remember to check against when additional PR updates get's merged